### ISSUE TITLE
Font size isn't applied to non-sticky/non-crit

### ISSUE THREAD

**Issue by Schaka** [AtheneGenesis/Vanilla_MikScrollingBattleText#14]
I know this is abandoned and I don't mind. But I spent a few hours going through the code and trying to find out where the issue is.
Everything normally attached to a scrolling area works fine.

Fonts for normal stuff just doesn't change at all. I can't make the font bigger or smaller.
I checked if the FontObject TextHeight is being set everywhere. Everything seems correct for me. 
Texture is also scaling correctly.

Can you point me towards where to look?

**Comment by AtheneGenesis** [comment]
Hi, there's a safety in [DisplayMessage()](https://github.com/AtheneGenesis/Vanilla_MikScrollingBattleText/blob/master/MikScrollingBattleText/MikScrollingBattleText.lua#L2750) to override a Fontsize that would be too big or too small but it may not longer be relevant with nowadays screen sizes.
At a glance I'd say the issue could be in MikSBT.DisplayMessage() or MikSBT.AddAnimation() but I haven't used this code in years.

Is this an issue from within MSBT or you're using the public function to send your notifications through MSBT ?

**Comment by Schaka** [comment]
It's a problem for what happens from within the addon itself.
I suspect it's because I'm playing on a 4k display.

I managed to solve it earlier today. It seems the initial size passed to SetFont isn't applied at all. Once you call SetTextHeight, it's also ignored UNLESS it's different from the original font size. I hackfixed this by applying an initial font size of fontSize-1 and then applying SetTextHeight with the correct font size right after. That took me forever to debug.

I'll submit a PR.
